testing/deployer/sprawl/catalog.go

```diff
@@ -286,7 +286,9 @@ func (s *Sprawl) syncWorkloadsForDataplaneInstances(cluster *topology.Cluster) e
 			} else {
 				syncWorkload = deregisterWorkloadFromNode
 			}
-			syncWorkload(node, wrk)
```

Comment:
bug1: the error was being ignored. when printed the error was one of "this v2 resource name is too long!"

(see bug2)